server/meshmodel/kubernetes/relationships/edge_mount.json

```diff
@@ -93,5 +93,55 @@
         }
       ]
     }
-  }]
+  },
+  {
+    "deny": {
+      "from": [],
+      "to": []
+    },
+    "allow": {
+      "from": [
+        {
+          "kind": "Pod",
+          "model": "kubernetes",
+          "match": {
+            "self": [
+              [
+                "settings",
+                "spec",
+                "serviceAccountName"
+              ]
+            ],
+            "ServiceAccount": [
+              [
+                "name"
+              ]
+            ]
+          }
+        }
+      ],
+      "to": [
+        {
+          "kind": "Secret",
+          "model": "kubernetes",
+          "match": {
+            "self": [
+              [
+                "name"
+              ]
+            ],
+            "ServiceAccount": [
+              [
+                "settings",
+                "secrets",
+                "_",
```

Comment:
@RipulHandoo what does `-` mean?

Reply:
Item. 
It's similar to its usage in [this example](https://github.com/meshery/meshery/blob/master/server/meshmodel/kubernetes/relationships/hierarchical_inventory.json#L116).

Reply:
Thanks, @RipulHandoo. Will you please answer my question with a link to our contrib docs that explains this?